rest-service/manager_rest/security/secured_resource.py

```diff
@@ -74,17 +74,29 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def missing_premium_feature_abort(func):
+def missing_premium_feature_abort():
+    """Throw a nicely formatted "premium only" error"""
+    abort_error(
+        error=MissingPremiumPackage(
+            'This feature exists only in the premium edition of Cloudify.'
+            '\nPlease contact sales for additional info.'
```

Comment:
Maybe add a link to our site, I think we done that for licensing

Reply:
not gonna change the message, it's unrelated to this pr